### DISCUSSION TITLE
Unexpected time conversion (end of month, instead of begin)

### DISCUSSION THREAD

**Discussion by franq77** [grafana/grafana-json-datasource#106] [Q&A] [answered]
My JSON response (From Cloudability API v3) returns as date dimension: "2021-02", this is expected. When I than apply the Time type:
![image](https://user-images.githubusercontent.com/17096183/117115359-51d77200-ad8d-11eb-8cf5-1b3c495f3207.png)

I get the following values, end of month time shifted 1h + DST (time filer is in UTC):

2021-01-31 23:00:00
2021-02-28 23:00:00
2021-03-31 22:00:00 <-- Summer time
2021-04-30 22:00:00

Instead of the expected, beginning of the month:
2021-01-01 00:00:00
2021-02-01 00:00:00
2021-03-01 00:00:00
2021-03-01 00:00:00

Is there a way to control this ? Or work around so that I get the beginning of the month?

**Comment by marcusolsson** [comment, marked as answer]
If there's no time zone defined, as in "2021-02", the time gets parsed in local time. If you've configured your Grafana to display time in UTC, there's indeed likely going to be a difference. 

If you're asking whether you can set the default time zone, then no. Another possible solution would be to use the dashboard time zone for not only displaying time, but to parse it as well.
